src/Microsoft.ML.Legacy/CSharpApi.cs

```diff
@@ -14444,7 +14444,7 @@ public MissingValuesRowDropperPipelineStep(Output output)
 
     namespace Legacy.Transforms
     {
-        public enum NAReplaceTransformReplacementKind
+        public enum NAReplaceTransformReplacementKind : byte
```

Comment:
> : byte [](start = 53, length = 7)

fix codegen? Or is it already good?

Reply:
it's already good

---
In reply to: [218248180](https://github.com/dotnet/machinelearning/pull/917#discussion_r218248180) [](ancestors = 218248180)